### PR TITLE
Fix the crash when no bookmark button on the menu bottom bar

### PR DESCRIPTION
**Pull request by proevan**
- Fix the crash when no bookmark button on the menu bar

Steps:
1. config the bottom bar with a bookmark button, and with no bookmark button on the menu bottom bar.
2. click bookmark button while browsing web page
3. app crash

- added `focusedSessionUrl` as a LiveData in `ChromeViewModel`

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/activity/MainActivity.java [hunk 1]
```diff
@@ -227,15 +230,9 @@ protected void onCreate(Bundle savedInstanceState) {
...
 
-        BookmarkViewModel.Factory factory = new BookmarkViewModel.Factory(
-                BookmarkRepository.getInstance(BookmarksDatabase.getInstance(this)));
-
-        bookmarkViewModel = ViewModelProviders.of(this, factory).get(BookmarkViewModel.class);
 
         PreferenceManager.getDefaultSharedPreferences(this)
                 .registerOnSharedPreferenceChangeListener(this);
-
-        downloadIndicatorViewModel = Inject.obtainDownloadIndicatorViewModel(this);
```
Comment (pharspher) [inline app/src/main/java/org/mozilla/focus/activity/MainActivity.java]: Where did this go?

File: app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java [hunk 2]
```diff
@@ -1288,6 +1288,7 @@ public void onSecurityChanged(@NonNull Session session, boolean isSecure) {
 
         @Override
         public void onUrlChanged(@NonNull Session session, @Nullable String url) {
+            chromeViewModel.onFocusedSessionChanged(url);
```
Comment (pharspher) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: calling onFocusedSEssionChanged() in onUrlChanged() seems weird to me. Maybe there's a better naming for onFocusedSessionChanged

File: app/src/main/java/org/mozilla/focus/activity/MainActivity.java (hunk 1 above)
Comment (proevan) [reply app/src/main/java/org/mozilla/focus/activity/MainActivity.java]: it's duplicate

File: app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java (hunk 2 above)
Comment (proevan) [reply app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: updated to onFocusedUrlChanged

**Pull request merged**